### ISSUE TITLE
Converted keras model has different parameters

### ISSUE THREAD

**Issue by Mikezhang97** [nerox8664/pytorch2keras#127]
The original PyTorch model has no bias (only weights) in certain Convo layers but the converted Keras model does have bias respectively. And the Keras model shows no BatchNorm2D layers compared to the PyTorch model. I am thinking that the code has conducted BN folding. Is there a way to switch off this function so it will be easier to match the weights?

**Comment by Tailwhip** [comment]
Have the same problem. Any updates?
